ui/apps/platform/cypress/integration/integrations/notifiers.test.js

```diff
@@ -247,7 +258,16 @@ describe('Notifier Integrations', () => {
             cy.get(selectors.buttons.test).should('be.disabled');
             cy.get(selectors.buttons.save).should('be.disabled');
 
-            // Step 2, check fields for invalid formats
+            // Step 2, check conditional fields
+
+            // Step 2.1, enable workload identity, this should remove the service account field
+            getInputByLabel('Use workload identity').click();
+            cy.get('[id=notifier.cscc.serviceAccount]').should('not.exist');
```

Comment:
Here is an alternative that asserts visible text:
```js
            cy.get(
                `.pf-c-form__group:has('.pf-c-form__control:contains("Service Account Key (JSON)")') textarea`
            ).should('not.exist');
```

Reply:
For your info, Stephan will merge changes that overlap with yours to fix ocp test failures:
https://github.com/stackrox/stackrox/pull/9076/files